backend-lambda/validation/validation.js

```diff
@@ -2,7 +2,7 @@ function validateProductId(productId) {
   if (!productId) {
     throw new Error("No product ID in path");
   }
-  return;
+  return parseInt(productId);
```

Comment:
We also need to check if it was successful. JS has this type called `NaN` (not a number). parseInt works kinda funny.
`parseInt('11')` = 11
`parseInt('eleven')` = NaN
`parseInt('11eleven)` = 11

You can use the built-in function `isNaN(arg)` to see if something is NaN or not.